app/javascript/packages/components/select-input.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef } from 'react';
+import type { InputHTMLAttributes, ForwardedRef } from 'react';
+import { useInstanceId } from '@18f/identity-react-hooks';
+
+export interface SelectInputProps extends InputHTMLAttributes<HTMLSelectElement> {
```

Comment:
what is this component used for?

Reply:
This will be used for the state ID dropdown on an alternative version of the PO search page.